charts/cadence/values.yaml

```diff
@@ -1,147 +1,665 @@
+# -- Provide a name to override the name of the chart
+nameOverride: ~
+# -- Provide a name to override the full names of resources
+fullnameOverride: ~
 
-log:
-  stdout: true
-  level: "debug"
+# Global configuration
+global:
+  # -- Global image configuration (shared only by Cadence Server services [frontend, worker, matching and history])
+  image:
+    repository: "docker.io/ubercadence/server"
+    tag: "v1.2.16-auto-setup"
```

Comment:
please update default image tag to `v1.3.1-auto-setup`. it comes with pre-registered domain "default" which simplifies getting started

Reply:
sounds good